### PR TITLE
Synchronize preface writer with persons

### PR DESCRIPTION
**Pull request by miri**
#184

### CODE REVIEW

File: haskala/modules/custom/haskala_migrate/includes/haskala_migrate.entity.inc [hunk 1]
```diff
@@ -75,13 +75,23 @@ abstract class HaskalaMigration extends Migration {
...
+          ->propertyCondition('type', 'person')
+          ->propertyCondition('title', $name)
+          ->execute();
+      }
+      // Try to search by its German name or hebrew name.
+      else {
+        $query = new EntityFieldQuery();
+        $result = $query->entityCondition('entity_type', 'node')
+          ->propertyCondition('type', 'person')
+          ->propertyCondition('title', '%'.$title.'%', 'LIKE')
```
Comment (bitamar) [inline haskala/modules/custom/haskala_migrate/includes/haskala_migrate.entity.inc]: Add space around dots

File: haskala/modules/custom/haskala_migrate/includes/haskala_migrate.entity.inc [hunk 2]
```diff
@@ -75,13 +75,23 @@ abstract class HaskalaMigration extends Migration {
...
     // When not finding a person, try to search only for its German name.
-    if (empty($result['node']) && $bundle == 'person' && mb_strrchr($title, ' - ')) {
-      list($name) = explode(' - ', $title);
-      $query = new EntityFieldQuery();
-      $result = $query->entityCondition('entity_type', 'node')
-        ->propertyCondition('type', 'person')
-        ->propertyCondition('title', $name)
-        ->execute();
+    if (empty($result['node']) && $bundle == 'person') {
+      if (mb_strrchr($title, ' - ')) {
```
Comment (bitamar) [inline haskala/modules/custom/haskala_migrate/includes/haskala_migrate.entity.inc]: This is probably redundent

Comment (miri) [inline haskala/modules/custom/haskala_migrate/includes/haskala_migrate.entity.inc]: After checking, the code is necessary for some cases in migrate production producers with full name that appear in short title in person CSV.

**Comment by bitamar** [comment]
@miri - Please merge

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**